core/templates/components/forms/schema-based-editors/schema-based-float-editor.directive.html

```diff
@@ -1,7 +1,7 @@
 <span ng-model="$ctrl.localValue" ui-validate="'$ctrl.validate($value)'">
   <span ng-form="$ctrl.floatForm">
     <span ng-if="!expressionMode">
-      <input type="number" step="any" autocomplete="off" ng-model="$ctrl.localValue" class="form-control display-inline oppia-float-form-input protractor-test-float-form-input"
+      <input type="number" step="any" min="0" autocomplete="off" ng-model="$ctrl.localValue" class="form-control display-inline oppia-float-form-input protractor-test-float-form-input"
```

Comment:
Hi @ReshuKumari, have you checked where else this directive is used?
Also, I think we should have this as a configurable property so that, if in a particular instance of this directive, we want to allow negative values, it should be possible to do so.